sql/auth/sql_auth_cache.cc

```diff
@@ -2489,8 +2488,7 @@ bool acl_reload(THD *thd, bool mdl_locked) {
   old_acl_users = acl_users;
   old_acl_dbs = acl_dbs;
   old_acl_proxy_users = acl_proxy_users;
-  old_acl_restrictions = move(acl_restrictions);
-  old_acl_fast_lookup = acl_fast_lookup;
```

Comment:
Is this removal intended? This causes a memory leak because old_acl_fast_lookup is never set, so the old acl_fast_lookup is not deleted.

Reply:
I made an error while cherry-picking. https://github.com/facebook/mysql-5.6/pull/1307